Charon/filetypes/OpenPackagingConvention.py

```diff
@@ -172,11 +171,10 @@ def getStream(self, virtual_path: str) -> IO[bytes]:
             raise OPCError("Writing directly to a relationship file is forbidden.")
 
         if virtual_path.startswith(self._metadata_prefix):
-            return BytesIO(json.dumps(self.getMetadata(virtual_path[len(self.metadata_prefix):])).encode("UTF-8"))
+            return json.dumps(self.getMetadata(virtual_path[len(self._metadata_prefix):])).encode("UTF-8")
```

Comment:
Does the result of json.dumps() equals the type IO[bytes]? I would expect an type error here now?

Reply:
Strange, the typing says:
```
def getStream(self, virtual_path: str) -> IO[bytes]:
```
I think PyCharm is wrong in this case. Bug in PyCharm? What does mypy say?

Reply:
Actually it seems that a bytes is used in all the examples and implementations, so the typing is just wrong.

Reply:
Indeed PyCharm was wrong here. Some of the example still have 'wrong' typing though, because they're missing `# type: BinaryIO` (as io.BytesIO() doesn't resolve to that apparently).

Reply:
So should this be changed or stay as is?

Reply:
I'll change it back to prevent breaking changes.